### PR TITLE
Add IPv4 oversized mtu test

### PR DESCRIPTION
**Pull request by SerhiyBoikoPLV**
Signed-off-by: Serhiy Boiko <serhiy.boiko@plvision.eu>

### CODE REVIEW

File: DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/ipv4/test_ipv4_mtu.py [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+        (ports[3], tg_ports[3], "4.4.4.1", "4.4.4.2", 24),
+    )
+    tg_to_swp_map = {
+        tg: swp for swp, tg, _, _, _ in address_map
+    }
+
+    # 2. Configure ports up
+    out = await IpLink.set(input_data=[{dent: [
+        {"device": port, "operstate": "up"}
+        for port, _, _, _, _ in address_map
```
Comment (yuriilisovskyi) [inline DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/ipv4/test_ipv4_mtu.py:64]: You can use:
`for port, *_ in address_map`

Comment (SerhiyBoikoPLV) [reply DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/ipv4/test_ipv4_mtu.py:64]: is this a python3.6 feature? If this is a 3.6+ feature then we can't use it

Comment (yuriilisovskyi) [reply DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/ipv4/test_ipv4_mtu.py:64]: I tried it on Python 3.5 and it did work, so I assume it should work.

Comment (SerhiyBoikoPLV) [reply DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/ipv4/test_ipv4_mtu.py:64]: Good. I'll update my commits.

**Pull request closed without merging**